help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,114 @@
+// ====------ onedpl_test_group_load.cpp---------- -*- C++ -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+#include<oneapi/dpl/iterator>
+
+
+bool helper_validation_function(const int* ptr,const char* func_name){
+  // Used for validation of output and expected output sequences
+  oneapi::dpl::counting_iterator<int> expected(0);
+  
+  for (int i = 0; i < 512; ++i) {
+    if (ptr[i] != expected[i]) {

```

Comment:
Seems like we don't really need to use a counting iterator here. I think we can just do `ptr[i] != i`.

Reply:
Yes changed